### DISCUSSION TITLE
Changing colors is hard

### DISCUSSION THREAD

**Discussion by CsatariGergely** [google/docsy#1846] [Q&A]
### Environment

- [X] Version of Docsy you are using: HEAD
- [X] How are you using Docsy? As a:
  - [X] Hugo module
  - [ ] NPM module
  - [ ] Git submodule
  - [ ] Other:
- [X] Version of Hugo you are using (output of `hugo version`): hugo v0.120.4-f11bca5fec2ebb3a02727fb2a5cfb08da96fd9df+extended linux/amd64 BuildDate=2023-11-08T11:18:07Z VendorInfo=gohugoio
- [X] OS:
  - [X] Linux
  - [ ] macOS
  - [ ] Windows
  - [ ] Other:

### Problem

There is not a central place where the used colors are defined and documentation does not help to find out how the colors can be changed.

**Comment by fekete-robert** [comment]
I think you are looking for this: https://getbootstrap.com/docs/5.0/customize/color/

**Reply by CsatariGergely** [reply]
Maybe, I'm not sure how to apply this to my site ;) I need to play a bit more with this.

**Comment by chalin** [comment]
Hi @CsatariGergely - can you be more specific in terms of the colors you want to change? I'm going to move this to discussions until we pin point more precisely what is needed. Btw, I just created the following issue:

- #1845

Maybe this is what you need?

**Reply by CsatariGergely** [reply]
These: 
![image](https://github.com/google/docsy/assets/11458775/c8e6de7b-c34e-420b-ac0f-6202273dc751)

**Reply by chalin** [reply]
Thanks for clarifying: those two colors are `$primary` and `$secondary` colors. For guidance on how to change them, see https://www.docsy.dev/docs/adding-content/lookandfeel/#site-colors:

> To customize your site's colors, add SCSS variable overrides to `assets/scss/_variables_project.scss`. For example, you can set the primary and secondary site colors as follows:
> 
> ```scss
> $primary: #390040;
> $secondary: #A23B72;
> ```

**Reply by CsatariGergely** [reply]
Uh-oh this works. Thanks. I just put the `_variables_project.scss.scss` to the wrong place in my previous attempts. 
I take it all back. Thanks for the help @chalin
